.pmd/exclude.properties

```
@@ -0,0 +1,156 @@
+org.apache.maven.impl.resolver.LocalSnapshotMetadataGenerator=UnusedFormalParameter
+org.apache.maven.impl.resolver.scopes.Maven3ScopeManagerConfiguration=TooManyStaticImports,UnnecessaryFullyQualifiedName
+org.apache.maven.impl.resolver.scopes.Maven4ScopeManagerConfiguration=TooManyStaticImports,UnnecessaryFullyQualifiedName
+org.apache.maven.internal.aether.DefaultRepositorySystemSessionFactory=EmptyControlStatement
+org.apache.maven.internal.impl.CleanLifecycle=UnnecessaryFullyQualifiedName
+org.apache.maven.internal.impl.DefaultLifecycle=UnnecessaryFullyQualifiedName
+org.apache.maven.internal.impl.DefaultLifecycleRegistry=UnnecessaryFullyQualifiedName,TooManyStaticImports
+org.apache.maven.internal.impl.DefaultTypeRegistry=CollapsibleIfStatements
+org.apache.maven.internal.impl.SiteLifecycle=UnnecessaryFullyQualifiedName
+org.apache.maven.internal.transformation.impl.DefaultConsumerPomArtifactTransformer=UnusedPrivateMethod
+org.apache.maven.internal.transformation.impl.DefaultConsumerPomBuilder=UnusedFormalParameter
+org.apache.maven.internal.xml.DefaultXmlService=UnusedPrivateMethod
+org.apache.maven.jline.DefaultPrompter=UselessParentheses
+org.apache.maven.jline.MessageUtils=UnnecessaryFullyQualifiedName
+org.apache.maven.lifecycle.DefaultLifecycles=UnusedFormalParameter
+org.apache.maven.lifecycle.LifecycleExecutionException=UnusedFormalParameter
+org.apache.maven.lifecycle.internal.DefaultLifecycleExecutionPlanCalculator=UselessParentheses
+org.apache.maven.lifecycle.internal.DefaultLifecycleTaskSegmentCalculator=UselessParentheses
+org.apache.maven.lifecycle.internal.MojoExecutor=UnusedLocalVariable
+org.apache.maven.lifecycle.internal.builder.BuilderCommon=EmptyControlStatement
+org.apache.maven.lifecycle.internal.concurrent.BuildContext=UnusedLocalVariable,CollapsibleIfStatements,EmptyControlStatement
+org.apache.maven.lifecycle.internal.concurrent.BuildPlanExecutor=TooManyStaticImports,UselessParentheses
+org.apache.maven.lifecycle.internal.concurrent.ConcurrentLifecycleStarter=UselessParentheses
+org.apache.maven.logging.LoggingExecutionListener=UselessParentheses
+org.apache.maven.model.building.DefaultModelBuilder=UnnecessaryFullyQualifiedName
+org.apache.maven.model.inheritance.DefaultInheritanceAssembler=UselessParentheses
+org.apache.maven.model.io.xpp3.MavenXpp3ReaderEx=UselessOverridingMethod
+org.apache.maven.model.io.xpp3.MavenXpp3WriterEx=UselessOverridingMethod
+org.apache.maven.model.merge.MavenModelMerger=CollapsibleIfStatements
+org.apache.maven.model.merge.ModelMerger=CollapsibleIfStatements
+org.apache.maven.model.validation.DefaultModelValidator=UnusedFormalParameter
+org.apache.maven.plugin.CycleDetectedInPluginGraphException=UnusedFormalParameter
+org.apache.maven.plugin.PluginConfigurationException=UnusedPrivateField
+org.apache.maven.plugin.PluginParameterException=UselessParentheses
+org.apache.maven.plugin.PluginParameterExpressionEvaluator=UselessParentheses,EmptyControlStatement
```

```suggestion
org.apache.maven.plugin.PluginParameterExpressionEvaluator=EmptyControlStatement
```
https://docs.openrewrite.org/recipes/staticanalysis/unnecessaryparentheses